packages/core/src/layout/Page/Page.stories.tsx

```diff
@@ -36,7 +36,7 @@ import {
 import { Box, Typography, Link, Chip, Grid } from '@material-ui/core';
 
 export default {
-  title: 'Example Plugin',
+  title: 'Layout/Example Plugin',
```

Comment:
This should be in Plugins, no?

Reply:
I saw it more as an example using our different core layout components. And "Plugins" would be stories defined in actual plugins.